core/domain/training_job_result_services.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions for training job result domain object and protobuf message."""
+
+from core.classifiers import algorithm_registry
+from core.domain.proto import training_job_response_payload_pb2
+
+def get_proto_message_from_training_job_result(job_result):
+    """Generate TrainingJobResult protobuf object from the TrainingJobResult
+    domain object.
+
+    Args:
+        job_result: TrainingJobResult. TrainingJobResult domain object
+            containing training job result data.
+
+    Returns:
+        TrainingJobResult protobuf object. Protobuf object corresponding to
+            TrainingJobResult protobuf message definition.
+    """
+    job_result.validate()
+    proto_message = (
+        training_job_response_payload_pb2.TrainingJobResponsePayload.JobResult()) # pylint: disable=line-too-long
```

Comment:
Could we import TrainingJobResponsePayload directly to minimize the length of the line?

Reply:
Sure, however, we always import the module and not a class or function defined within a module throughout Oppia and Oppia-ML. I always thought of it as a coding style adopted by Oppia.

Reply:
We might want to change that pattern since otherwise these proto lines are going to be super long. @seanlip do you have a preference on the existing convention?

Reply:
I believe you can break before one of the dots. See e.g. line 257 in core/controllers/creator_dashboard.py.

@prasanna08 try it and see?

Reply:
It worked! Thanks.